K8s-deployment/Charts/postgresql/install.sh

```diff
@@ -7,8 +7,8 @@ kubectl create secret generic pgpool-auth  --from-file=./secrets/passwords/usern
 kubectl create configmap init-scripts --from-file=./init-scripts/ -n postgres
 sleep 10
 # install postgres asynchronous cluster  
-helm repo add bitnami https://raw.githubusercontent.com/bitnami/charts/43174e1970616584f893f66c1fbcac00d110c633/bitnami/
-helm install -f psql-async-values.yaml -f resource-values.yaml psql --version 7.7.3 $@ bitnami/postgresql-ha -n postgres
+helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
```suggestion
helm repo add bitnami https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
```
Better to use the bitnami full index